course3/3.some_projects/pre-proj-geo/user/internal/controller/grpc/proto/user.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
```

Comment:
proto files are usually located in directory
`project-root/api/proto`
here are the examples
https://github.com/golang-standards/project-layout/tree/master/api